src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -505,6 +505,16 @@ private int WriteRunToErrorList(Run run, string logFilePath)
                 }
             }
 
+            if (run.HasAbsentResults())
```

Comment:
>if [](start = 12, length = 2)

This is the relevant code, together with the implementations of the "stub" table data sources and their base class